create-db-ddl.sql

```diff
@@ -1397,8 +1396,10 @@ CREATE TYPE cqc."UserAuditChangeType" AS ENUM (
     'passwdReset',
     'loginSuccess',
     'loginFailed',
-    'loginWhileLocked'
+    'loginWhileLocked',
+    'deletedUser'
```

Comment:
Establishment and Worker audit already have "deleted". Can this User audit event type also be simply `deleted` for consistency? The "User" suffix is superfluous because the event type is already constrained by `cqc.UserAuditChangeType`

```
-- Type: EstablishmentAuditChangeType

-- DROP TYPE cqc."EstablishmentAuditChangeType";

CREATE TYPE cqc."EstablishmentAuditChangeType" AS ENUM
    ('created', 'updated', 'saved', 'changed', 'deleted', 'wdfEligible', 'overalWdfEligible');

ALTER TYPE cqc."EstablishmentAuditChangeType"
    OWNER TO sfcadmin;
```

Reply:
I think deletedUser is based off what the trello card said, can change to "deleted" if needs be

Reply:
It's more consistent with the existing audit event types to be simply `delete`. I can't see on the trello card any specific task to update the database schema's type, so I can't see how it refers to `deleteUser`.

Reply:
Its this one on the 1st task: "and creates a User audit event `deletedUser`.". Will change to delete.